packages/hawtio/jest.config.ts

```diff
@@ -10,7 +10,7 @@ export default {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$':
       '<rootDir>/src/__mocks__/fileMock.js',
     '\\.(css|less)$': '<rootDir>/src/__mocks__/styleMock.js',
-    '@hawtio/(.*)': '<rootDir>/src/$1',
+    '@hawtiosrc/(.*)': '<rootDir>/src/$1',
```

Comment:
hmm, we might need to revisit the usage of this custom path alias... for the time being it's ok, but we should reconsider if we really need this or not and if we do what's the proper name for the alias.

Reply:
Agree. It ended up bringing more trouble with the introduction of an additional package. Created https://github.com/hawtio/hawtio-next/issues/105.